circle.yml

```diff
@@ -10,13 +10,8 @@ dependencies:
     - make install
 test:
   override:
-    - make lint
-    - make test-all
-    - behave --junit --junit-directory $CIRCLE_TEST_REPORTS/behave/junit.xml:
+    - case $CIRCLE_NODE_INDEX in 0) make lint; make test-all; make docs ;; 1) behave integration-tests/ --junit --junit-directory $CIRCLE_TEST_REPORTS/behave/junit.xml ;; esac:
```

Comment:
Will increasing max retries (which I think we have an open PR for) help? (happy for it to go though in a later PR)